x/stake/keeper/validator_test.go

```diff
@@ -634,6 +634,7 @@ func TestGetTendermintUpdatesInserted(t *testing.T) {
 	require.Equal(t, validators[4].ABCIValidator(), updates[0])
 }
 
+//TODO why is this called NotValidatorCliff?
```

Comment:
No idea - per `git blame`, you wrote it.

Looks like this tests the cliff validator logic (in the case when we *do* have a cliff validator) - maybe `TestGetTendermintUpdatesWithCliffValidator`?

Reply:
haha - I wrote that TODO at some unruly hour knowing I might forget about it... thanks! - yeah cool I like your suggested name

Reply:
addressed